app/core/myeclpay/cruds_myeclpay.py

```diff
@@ -0,0 +1,971 @@
+    db: AsyncSession,
+) -> None:
+    transaction = models_myeclpay.Transaction(
+        id=transaction_id,
+        debited_wallet_id=debited_wallet_id,
+        debited_wallet_device_id=debited_wallet_device_id,
+        credited_wallet_id=credited_wallet_id,
+        transaction_type=transaction_type,
+        seller_user_id=seller_user_id,
+        total=total,
+        creation=creation,
+        status=status,
+        store_note=store_note,
+    )
+    db.add(transaction)
+
+
+async def update_transaction_status(
+    transaction_id: UUID,
+    status: TransactionStatus,
+    db: AsyncSession,
+) -> None:
+    await db.execute(
+        update(models_myeclpay.Transaction)
+        .where(models_myeclpay.Transaction.id == transaction_id)
+        .values(status=status),
+    )
+
+
+async def get_transaction(
+    transaction_id: UUID,
+    db: AsyncSession,
+) -> schemas_myeclpay.Transaction | None:
+    result = (
+        (
+            await db.execute(
+                select(models_myeclpay.Transaction).where(
+                    models_myeclpay.Transaction.id == transaction_id,
+                ),
+            )
+        )
+        .scalars()
+        .first()
+    )
+    return (
+        schemas_myeclpay.Transaction(
+            id=result.id,
+            debited_wallet_id=result.debited_wallet_id,
+            credited_wallet_id=result.credited_wallet_id,
+            transaction_type=result.transaction_type,
+            seller_user_id=result.seller_user_id,
+            total=result.total,
+            creation=result.creation,
+            status=result.status,
+        )
+        if result
+        else None
+    )
+
+
+async def get_transactions(
+    db: AsyncSession,
+) -> Sequence[schemas_myeclpay.Transaction]:
+    result = await db.execute(select(models_myeclpay.Transaction))
+    return [
+        schemas_myeclpay.Transaction(
+            id=transaction.id,
+            debited_wallet_id=transaction.debited_wallet_id,
+            credited_wallet_id=transaction.credited_wallet_id,
+            transaction_type=transaction.transaction_type,
+            seller_user_id=transaction.seller_user_id,
+            total=transaction.total,
+            creation=transaction.creation,
+            status=transaction.status,
+        )
+        for transaction in result.scalars().all()
+    ]
+
+
+async def get_transactions_by_wallet_id(
+    wallet_id: UUID,
+    db: AsyncSession,
+) -> Sequence[models_myeclpay.Transaction]:
+    result = await db.execute(
+        select(models_myeclpay.Transaction)
+        .where(
+            or_(
+                models_myeclpay.Transaction.debited_wallet_id == wallet_id,
+                models_myeclpay.Transaction.credited_wallet_id == wallet_id,
+            ),
+        )
+        .options(
+            selectinload(models_myeclpay.Transaction.debited_wallet),
+            selectinload(models_myeclpay.Transaction.credited_wallet),
+        ),
+    )
+    return result.scalars().all()
```

Comment:
Shouldn't we used schemas everywhere or nowhere? It's a bit confusing when similar cruds use schemas and others don't.
```suggestion
    return [
        schemas_myeclpay.Transaction(
            id=transaction.id,
            debited_wallet_id=transaction.debited_wallet_id,
            credited_wallet_id=transaction.credited_wallet_id,
            transaction_type=transaction.transaction_type,
            seller_user_id=transaction.seller_user_id,
            total=transaction.total,
            creation=transaction.creation,
            status=transaction.status,
        )
        for transaction in result.scalars().all()
    ]
```